apps/test/netsim/NetSimLocalClientNode.js

```diff
@@ -38,6 +38,62 @@ describe("NetSimLocalClientNode", function () {
     assert(undefined !== testRemoteNode, "Made a remote node");
   });
 
+  describe("onWireTableChange_", function () {
+    it ("detects when remote client disconnects, and removes local wire", function () {
+
+      var localWireRow, remoteWireRow;
+
+      testLocalNode.connectToNode(testRemoteNode, function () {
+        testRemoteNode.connectToNode(testLocalNode, function () {
```

Comment:
FYI, in tests you can express these sequentially if you want to (as all remote operations are synchronous against the fake shard), just passing empty functions as callbacks.  It might actually be _more_ accurate, as running these callbacks during the call to the wrapper function can sometimes have bad side effects.